cypress/helpers/stories.ts

```diff
@@ -24,7 +24,7 @@ declare global {
  */
 export function load(categorization: string, story: string) {
   cy.window().then(win => {
-    win.setCurrentStory(categorization, story);
+    win.setCurrentStory(categorization.toLowerCase(), story.toLowerCase());
```

Comment:
`toLowerCase` allows us to use the same string as we find in `.add`. Easier for copy/paste